oxide.go

```diff
@@ -481,11 +481,7 @@ func (d *Driver) SetConfigFromFlags(opts drivers.DriverOptions) error {
 		errRequiredFlag = errors.Join(errRequiredFlag, errors.New(errRequiredOptionNotSet+flagBootDiskImageID))
 	}
 
-	if errRequiredFlag != nil {
-		return errRequiredFlag
-	}
-
-	return nil
+	return errRequiredFlag
```

Comment:
`errRequiredFlag` will be nil if there were no flag errors, so we shouldn't need the nil check and return nil